packages/mobile/src/components/core/Text.tsx

```diff
@@ -23,7 +24,10 @@ export type TextProps = RNTextProps & {
   fontSize?: FontSize | 'inherit'
 }
 
-const useStyles = makeStyles(
+const useStyles = makeStyles<
+  Pick<TextProps, 'noGutter' | 'weight' | 'fontSize'> &
+    NestedNonNullableNonOptional<Pick<TextProps, 'variant' | 'color'>>
```

Comment:
so what about just `Required<Pick<TextProps, 'variant' | 'color'>>` since i think variant and color are not nullable?